VMRecovery/Testing/Create-TestVM.ps1

```diff
@@ -0,0 +1,848 @@
+<#
+.SYNOPSIS
```

Comment:
Great to see some test automation.  When ready, you might consider if it can be wired into TravisCI so that it runs automatically with future check-ins/builds.

Reply:
This is good information, we will look into this but for right now we have removed the testing folder from the repo.